config/initializers/ood.rb

```diff
@@ -0,0 +1,9 @@
+Rails.application.configure do
+  config.x.ood.nav_categories = ENV['OOD_NAV_CATEGORIES'] ? ENV['OOD_NAV_CATEGORIES'].split(",") : ["Files", "Jobs", "Clusters", "Desktops"]
+
+  #TODO: determine how to configure this via ENV
+  projects = User.new.groups.map(&:name).grep(/^P./)
+  config.x.ood.files_other_paths = projects.map { |p| Pathname.new("/fs/project/#{p}") }
+  config.x.ood.files_other_paths << Pathname.new("/fs/scratch/#{User.new.name}")
+  config.x.ood.files_other_paths << Pathname.new("/fs/scratch/#{projects.first}")
```

Comment:
Instead of doing this, we could specify colon delimited files paths in an `OOD_FILES_PATHS` env var and use bash:

```
OOD_FILES_PATHS=`groups | ruby -e 'puts ARGF.read.scan(/(P\S*)/).map { |p| "/fs/project/#{p}:/fs/scratch/#{p}"}.join(":")'`
OOD_FILES_PATHS=$OOD_FILES_PATHS:/fs/scratch/$USER
```

Not sure if there is a more clever or succinct way to do this in bash.